test/common_utils.py

```diff
@@ -295,23 +291,21 @@ def __init__(
         if isinstance(expected, PIL.Image.Image):
             expected = to_image(expected)
 
-        # Convert CV-CUDA tensors to torch.Tensor (regardless of what the other is)
-        try:
-            import cvcuda
-            from torchvision.transforms.v2.functional import cvcuda_to_tensor
+        if _is_cvcuda_available():
+            cvcuda = _import_cvcuda()
 
             if isinstance(actual, cvcuda.Tensor):
-                actual = cvcuda_to_tensor(actual)
+                actual = cvcuda_to_tensor(actual)  # No import needed here anymore!
                 # Remove batch dimension if it's 1 for easier comparison
                 if actual.shape[0] == 1:
                     actual = actual[0]
+                actual = actual.cpu()
             if isinstance(expected, cvcuda.Tensor):
                 expected = cvcuda_to_tensor(expected)
                 # Remove batch dimension if it's 1 for easier comparison
                 if expected.shape[0] == 1:
                     expected = expected[0]
-        except ImportError:
-            pass
+                expected = expected.cpu()
```

Comment:
Are we able to split the logic to drop batch and move to cpu into a helper function? We do the same thing in the test itself, and I think it would improve clarity to have an explicit helper. 

[Line in reference](https://github.com/pytorch/vision/pull/9277/commits/1e5e9ede063a415b0b48084d02e68191dd3ac25e#diff-9c2dde92db86c123fee225e39b7c1ef96e08a3e79a9dcc9a2d68b21ed51a81d0L1313)